x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -384,3 +384,7 @@ func TestQueryConsumerIdFromClientId(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expectedConsumerId, res.ConsumerId)
 }
+
+func TestQueryProposedConsumerChainIDs(t *testing.T) {
+	// TODO (PERMISSIONLESS)
```

Comment:
@mpoke This might get quite tricky because we need to mock governance, proposals, etc. I wonder if we should go back to a solution where we store the association between `consumerId` to a `proposalId` and then just implement the `QueryProposedConsumerChainIDs` by looking at that.

Reply:
This test was removed.